### PR TITLE
feat(TMRX-1225): update leadStory component to accommodate slice

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
- Allow for content to be placed below image
- Allow for different Headline sizes
 
[TMRX-1225](https://nidigitalsolutions.jira.com/browse/TMRX-1225)

[TMRX-1225]: https://nidigitalsolutions.jira.com/browse/TMRX-1225?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/components/slices/lead-story/index.tsx [hunk 1]
```diff
@@ -25,8 +25,9 @@ export interface LeadStoryProps {
   image: string;
   url: string;
   articleType?: string;
+  hasTagOrTimeToRead?: boolean;
+  isBukcet1?: boolean;
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/lead-story/index.tsx:29]: typo - `isBucket1`

File: packages/ts-newskit/src/components/slices/lead-story/index.tsx [hunk 2]
```diff
@@ -36,7 +37,9 @@ export const LeadStory = ({
   caption,
   image,
   url,
-  articleType
+  articleType,
+  hasTagOrTimeToRead,
+  isBukcet1
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/lead-story/index.tsx:42]: as above

File: packages/ts-newskit/src/components/slices/lead-story/index.tsx [hunk 3]
```diff
@@ -109,7 +119,7 @@ export const LeadStory = ({
         >
           {summary}
         </TextBlock>
-        <Visible md lg xl>
+        <Visible xs={hasTagOrTimeToRead} sm={hasTagOrTimeToRead} md lg xl>
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/lead-story/index.tsx:122]: nice!

**Review by douglasmik — APPROVED** [review]

File: packages/ts-newskit/src/components/slices/lead-story/index.tsx (hunk 1 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/lead-story/index.tsx]: fixed

File: packages/ts-newskit/src/components/slices/lead-story/index.tsx (hunk 2 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/lead-story/index.tsx]: fixed

**Pull request merged**